tokio-postgres/tests/test/types/mod.rs

```diff
@@ -19,10 +19,10 @@ mod bit_vec_06;
 mod chrono_04;
 #[cfg(feature = "with-eui48-0_4")]
 mod eui48_04;
-#[cfg(feature = "with-geo-types-0_4")]
```

Comment:
I did remove the 0.4 integration from the test module. It seemed unused/vestigial.

Reply:
Oops, yeah thanks that was a holdover from a previous release.